### PR TITLE
Feat/new fields metadata

### PR DESCRIPTION
**Pull request by a-crea**
This pull request includes the new metadata fields in list view, detail view, new and edit views. It also contains the new functionality to "ADD NEW" value to existing options.

### CODE REVIEW

**Comment by gappc** [comment]
@a-crea thank you for this PR.

Given that the PR changes 66 files, I took a deeper look at it. It seems, that most modifications should not be part of this PR and are related to other work - maybe PR #431 or even PR #429?

I would appreciate if you could clean up this PR and provide only the changes necessary for this PR to fulfill its requirements. Otherwise it gets really hard for me to do a proper review

thx

**Comment by a-crea** [comment]
@gappc Hi!
I have rebased both this PR and #431.

This PR should address #416. Shouldn't we take into account both the new input and the new fields in this PR? I believe it was already treated as a "bundle" since the two features are interconnected.

**Comment by gappc** [comment]
@a-crea thank you for your effort, but still it seems that something is odd.

If you take a look at the number of files this PR wants to merge on the top right of this page, you'll see that 78 files wait to be merged. I'm pretty sure that most of those files do not belong to this PR. I also tested to merge this PR locally (on my machine) into the current `development` branch, which resulted in at least one merge conflict (file .github/workflows/main.yml).

Would it be possible for you to open a new PR where you apply your changes (and only those)? I think that would be the easiest way to go forward.

thx

File: databrowser/src/domain/cellComponents/components/cells/customDataArrayCell/CustomDataArrayTable.vue [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    </template>
+    <template #noItems>No tags have been defined yet</template>
+    <template #addItems>
+      <EditListAddButton :text="'Add new'" @click="addItems([undefined])" />
+    </template>
+  </EditListTable>
+</template>
+
+<script setup lang="ts">
+import { computed, toRefs } from 'vue';
```
Comment (gappc) [inline databrowser/src/domain/cellComponents/components/cells/customDataArrayCell/CustomDataArrayTable.vue:41]: @a-crea please remove the import for `computed`, it is never used

Comment (a-crea) [reply databrowser/src/domain/cellComponents/components/cells/customDataArrayCell/CustomDataArrayTable.vue]: done!

**Comment by gappc** [comment]
@a-crea thank you for the `computed` import-fix, now please fix the build issue (https://github.com/noi-techpark/it.bz.opendatahub.databrowser/pull/432/files/30370103e87ba0b423f7dafa967e3685cf889cb7#diff-c700510ba9aafb210658d4b70f9db4320d846d0b1f47e9dba6e90b7bc3c4eb96) and then I'll be able to merge this PR

**Comment by a-crea** [comment]
@gappc done, let me know :)

**Comment by gappc** [comment]
@a-crea I've merged this PR, thx for your work :+1:

**Pull request merged**